business_requirement_deliverable/models/business.py

```diff
@@ -49,6 +60,40 @@ class BusinessRequirementResource(models.Model):
         string='Business Requirement Deliverable',
         ondelete='cascade'
     )
+    business_requirement_id = fields.Many2one(
+        comodel_name='business.requirement',
+        string='Business Requirement',
+        required=True,
+    )
+    business_requirement_partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        related='business_requirement_id.partner_id',
+        string='Business Requirement',
+        store=True
+    )
+    business_requirement_project_id = fields.Many2one(
+        comodel_name='project.project',
+        related='business_requirement_id.project_id',
+        string='Business Requirement',
+        store=True
+    )
+
+    @api.model
+    def fields_view_get(self, view_id=None,
```

Comment:
@sudhir-serpentcs Please do the more specif you can the PR. I wish to see that on separated PR and for the proper module.

And by they way this could be done IMO simpler like is actually done:

Draft Confirmed Approved SH Appr. In prog
Cust. Story V X X X X
Actually works as the requirement see the `states` param
```
    business_requirement = fields.Html(
        'Customer Story',
        readonly=True,
        states={'draft': [('readonly', False)]}
    )

```

Do still think need to overwrite `fields_view_get` ?
I would call it editable rules instead of access rules

cc @elicoidal